client/ui/contacts.go

```diff
@@ -102,3 +109,37 @@ func contactsData(index *core.QModelIndex, role int) *core.QVariant {
 func rowCount(parent *core.QModelIndex) int {
 	return listOfContacts.Size()
 }
+
+//
+func sendContactsRequestToServer() {
+	newMessageHeader := messageService.MessageHeader{
+		Type_:    config.GetConfig().MessageType.Contacts,
+		Command:  config.GetConfig().MessageCommand.ContactsRequest,
+		UserName: userinfo.CurrentUserInfo.UserName,
+		Token:    userinfo.CurrentUserInfo.Token,
+	}
+	//we have no data to send, so create empty Body
+	newMessageBody := messageService.MessageBody{}
+	newRawMessageBody, err := json.Marshal(&newMessageBody)
+	if err != nil {
+		loger.Log.Warningf("Can`t marshal contacts message body. %s", err)
+		qmlStatus.SendStatus("Can not load contacts.")
+		return
```

Comment:
It would be useful to return error. Imagine if error happens here,waht will user do without contacts?